share/ci/install/pypicongpu.sh

```diff
@@ -41,28 +41,26 @@ cd $CI_PROJECT_DIR
 micromamba create -n pypicongpu python=${PYTHON_VERSION} --ssl-verify false
 micromamba activate pypicongpu
 python3 --version
-MODIFIED_REQUIREMENT_TXT_PICMI=$CI_PROJECT_DIR/lib/python/picongpu/picmi/modified_requirements.txt
-MODIFIED_REQUIREMENT_TXT_PYPICONGPU=$CI_PROJECT_DIR/lib/python/picongpu/pypicongpu/modified_requirements.txt
-# The environment variables for changing the version of a dependency package are set globally.
-# Therefore, all calls of the requirements_txt_modfifier.py script are affected.
-# To avoid unexpected changes to dependencies, the list of ignored environment variables.
-python3 $CI_PROJECT_DIR/share/ci/install/requirements_txt_modifier.py \
-    -i $CI_PROJECT_DIR/lib/python/picongpu/picmi/requirements.txt \
-    -o $MODIFIED_REQUIREMENT_TXT_PICMI \
-    --ignore_env_args PYPIC_DEP_VERSION_referencing PYPIC_DEP_VERSION_jsonschema
-python3 $CI_PROJECT_DIR/share/ci/install/requirements_txt_modifier.py \
-    -i $CI_PROJECT_DIR/lib/python/picongpu/pypicongpu/requirements.txt \
-    -o $MODIFIED_REQUIREMENT_TXT_PYPICONGPU \
-    --ignore_env_args PYPIC_DEP_VERSION_pydantic PYPIC_DEP_VERSION_picmistandard
-
-echo "modified_requirements.txt: "
-cat $MODIFIED_REQUIREMENT_TXT_PICMI
-cat $MODIFIED_REQUIREMENT_TXT_PYPICONGPU
+# install requirements of pyproject_toml_modifier.txt
+pip3 install -r ${CI_PROJECT_DIR}/share/ci/install/pyproject_toml_modifier_requirments.txt
```

Comment:
```suggestion
pip3 install -r ${CI_PROJECT_DIR}/share/ci/install/pyproject_toml_modifier_requirements.txt
```
(Of course, also rename the file itself.)

Reply:
done